src/eei.cpp

```diff
@@ -671,6 +671,16 @@ inline int64_t maxCallGas(int64_t gas) {
       uint32_t dataOffset;
       uint32_t dataLength;
 
+      if (kind == EEICallKind::Call || kind == EEICallKind::CallCode) {
+        valueOffset = arguments[2].geti32();
+        dataOffset = arguments[3].geti32();
+        dataLength = arguments[4].geti32();
+      } else {
+        valueOffset = 0;
```

Comment:
It's better to always init variables. Then this is not needed.

Reply:
True, although they are implicitly initialised to 0 anyway. Here they are grouped, and I think it makes it clear that in this case value is ignored as option.

Reply:
The `valueOffset` is not initialized. I'm ok with leaving this assignment here.